src/node_alive/autostarter.py

```diff
@@ -0,0 +1,119 @@
+# ROS
+import os
+import roslaunch
+import rospkg
+import rospy
+from std_msgs.msg import String
+
+# TU/e Robotics
+from node_alive.srv import AutoStarterCommand, AutoStarterCommandRequest, AutoStarterCommandResponse
+
+class AutoStarter(object):
+    """
+    Launches a launch file and waits for a command to either shutdown or to restart.
+    It is also possible to start other launch files that are stored on the parameter server, like the demo's.
+    """
+    launch_files_list = []
+
+    def __init__(self):
+        """
+        Loads the launch files that can be started from the parameter server.
+        Creates a service over which the command will be received, a publisher that publishes the current launch file.
+        Set some variables.
```

Comment:
Idem. "Set some variables" also doesn't add anything

Reply:
removed in a1ef39d19ac8416def3517506f5d571fcb9e1645